bigquery_etl/view/__init__.py

```diff
@@ -353,6 +353,12 @@ def publish(self, target_project=None, dry_run=False):
                     print(f"Could not update field descriptions for {target_view}: {e}")
 
                 table = client.get_table(target_view)
+                if not self.metadata:
+                    return
+
+                table.description = self.metadata.description
+                table.friendly_name = self.metadata.friendly_name
+
```

Comment:
Does the below `update_table` call need to be updated? It explicitly mentions "labels" not sure if that needs to be done for `friendly_name` and `description`?